app/views/index.erb

```diff
@@ -67,13 +66,16 @@
                 <%= image_tag(get_image_url('map.jpg')) %>
               </a>
             </div>
-            <div class="whereinfo">
-              <p class="map-heart"><strong>Somewhere in Kolkata</strong> on April 21<sup>st</sup>
-                 and 22<sup>nd</sup>, 2015
+            <div class=whereinfo">
+              <p class="map-heart"><strong>Wedding : "SARENGI" </strong> SUKANTA NAGAR - 2nd LANE, Jessore Road South </br>P.O. Michael Nagar Kolkata - 700133,<strong>Wednesday 22<sup>nd</sup>, April'15</strong>
               </p>
-              <p class="map-time"><strong>Ceremony:</strong> @ TBD <br />
+              <p class="map-mehendi"><strong>Mehendi & Sangeet : "De Kutir" </strong>  - 202, Sukanata Sarani , B.T. College </br> P.O. New Barrackpur Kolkata - 700131,<strong>Tuesday 21<sup>st</sup>, April'15</strong>
+              </p>
+              <p class="map-time"><strong>Ceremony : </strong>Wedding @TBD<br /> Mehendi & Sangeet @TBD
+              </p>
+              <p class="map-driving"><strong>Directions : &nbsp;</strong><a href="https://www.google.co.in/maps/place/Sarengi,+Sarengi,+Sukanta+Nagar+2nd+Lane,+Madhyamgram,+Ward+No:18,+Kol-133./@22.677413,88.446309,17z/data=!3m1!4b1!4m2!3m1!1s0x39f89f1be70f83c9:0xe4a9d7054860b0e5" target="_blank">To the Banquet Hall</a>
+              </br> <a href="" target="_black">To my home! @De Kutir :) </a>
```

Comment:
Don't have hyperlinks that don't link anywhere, either update the target or remove the a tag.